app/javascript/application.js

```diff
@@ -1,8 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import '@patternfly/react-core/dist/styles/base.css';
```

Comment:
I'd suggest keeping it here so all of the future components will benefit from the PF styles, and not just the `FullArticle` component.